lib/v2/cankaoxiaoxi/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/:category?': ['yuxinliu-alex'],
```

Comment:
```suggestion
    '/news/:category': ['yuxinliu-alex'],
```
This should match `router.js`